cmd/ftl/cmd_init.go

```
@@ -36,13 +38,31 @@ type initGoCmd struct {
 	Name    string            `arg:"" help:"Name of the FTL module to create underneath the base directory."`
 }
 
+func isValidGoPackageName(name string) bool {
+	validNamePattern := `^[a-zA-Z][a-zA-Z0-9_]*$`
+	matched, _ := regexp.MatchString(validNamePattern, name)
```

Use `regexp.MustCompile()` instead of ignoring the error.